UICatalog/Scenarios/AdvancedClipping.cs

```diff
@@ -0,0 +1,166 @@
+﻿using System.Text;
+using System.Timers;
+using Terminal.Gui;
+
+namespace UICatalog.Scenarios;
+
+[ScenarioMetadata ("AdvancedClipping", "AdvancedClipping Tester")]
+[ScenarioCategory ("AdvancedClipping")]
+public class AdvancedClipping : Scenario
```

Comment:
> One question. When we click on a window in this scenario should that window brough to front?

No, not by default because they don't have Overlapped set. Use ArrangmentEditor to change them and you'll see.

Reply:
> Another question. If `Application.Top` is moveable then the background should be cleared or set it to non-moveable.

Correct. This is an outstanding bug. I plan on fixing it later.